### DISCUSSION TITLE
Set main window to full screen on startup from session file

### DISCUSSION THREAD

**Discussion by bernardassan** [kovidgoyal/kitty#3735] [Q&A] [answered]
Is there a way to start a terminal session in full screen from a session file.

**Comment by kovidgoyal** [comment, marked as answer]
No, use the command line option for that.

**Reply by bernardassan** [reply]
What is the command line option for setting full screen :confused: . I know it can be done with the default mapping `ctrl+shif+f11` but I know of no command line option

**Reply by martinetd** [reply]
`--start-as=fullscreen`

**Reply by bernardassan** [reply]
@martinetd  and @kovidgoyal  Thanks very much for your prompt reply. Using --start-as=fullscreen when starting kitty does what i want. Thank you guys soo much for creatings such an awesome Terminal Emulator. More grease to your elbows
